code/modules/mob/living/emote.dm

```diff
@@ -407,9 +418,17 @@
 	key = "sneeze"
 	key_third_person = "sneezes"
 	message = "sneezes."
+	audio_cooldown = 5
 	message_mime = "acts out an exaggerated silent sneeze."
 	emote_type = EMOTE_VISIBLE | EMOTE_AUDIBLE
 
+/datum/emote/living/sneeze/get_sound(mob/living/user)
+	var/mob/living/carbon/human/human_user = user
+	if (!HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
+		if(human_user.gender == FEMALE)
+			return 'sound/voice/human/female_sneeze1.ogg'
+		else return 'sound/voice/human/male_sneeze1.ogg'
```

Comment:
```suggestion
			return 'sound/voice/human/female_sneeze1.ogg'
		return 'sound/voice/human/male_sneeze1.ogg'
```
Don't need an else following a return.

Reply:
wouldn't that return 2 paths if the human_user is female? there is a check for if the user is female, then it returns the female_sneeze, otherwise it returns the male sneeze

Reply:
no because the return immediately halts the proc at that line, it won't continue to line 430 if we return at line 429

Reply:
oh right, thanks for the explanation!